InomialClient.js

```diff
@@ -7,12 +7,19 @@
 // subscriptions.
 //
 
+// Log level constants
+const NONE = 0;     // No logging at all.
+const ERRORS = 1;   // Only print errors (e.g. connection dropouts).
+const INFO = 2;     // Also print informational messages (connection establishment).
+const FINE = 3;     // Also print wire-level logging of GraphQL requests/responses.
+
 module.exports = {
-    connect: function(hostname, stage, origin, apikey) {
-        const client = new InomialClient(hostname, stage, origin, apikey);
+    connect: function(hostname, stage, origin, apikey, websocketClientConfig, logLevel) {
```

Comment:
instead of making the connect function more complex, consider exporting a setVerbose, setClientConfig etc function or property.

Reply:
Unfortunately I'm not able to create a `setClientConfig()` method, because in `InomialClient.js` you've only exported the function called `connect()` which creates a client instance as a local variable & then already begins making it connect and then returns it to the caller (w/o the caller having a chance to call a `setClientConfig()` method before initiating the connection to make the TLS parameters take effect).

The only viable way to achieve this at the moment is a fifth constructor argument.